game-engine/src/test/scala/MapObject_Lizard_Tests.scala

```diff
@@ -0,0 +1,45 @@
+import org.scalatest.FunSuite
+import test.TestHelper
+import za.co.entelect.challenge.game.contracts.Config.Config
+import za.co.entelect.challenge.game.contracts.command.RawCommand
+import za.co.entelect.challenge.game.contracts.commands.CommandFactory
+import za.co.entelect.challenge.game.contracts.game.CarGamePlayer
+
+class MapObject_Lizard_Tests extends FunSuite{
+    private val commandText = "NOTHING";
+    private var commandFactory: CommandFactory = null;
+    private var nothingCommand: RawCommand = null;
+
+    def initialise() = {
+        Config.loadDefault();
+        commandFactory = new CommandFactory;
+        nothingCommand = commandFactory.makeCommand(commandText)
+        nothingCommand.setCommand(commandText)
+    }
+
+    test("Given players during race when player hits lizard powerup then player gains 1 lizard powerup") {
+        initialise()
+        val gameMap = TestHelper.initialiseGameWithMapObjectAt(1, 3, Config.LIZARD_MAP_OBJECT);
+        val testGamePlayer1 = TestHelper.getTestGamePlayer1();
+
+        val testCarGamePlayer = testGamePlayer1.asInstanceOf[CarGamePlayer];
+        testCarGamePlayer.speed = Config.SPEED_STATE_2
+
+        TestHelper.processRound(gameMap, nothingCommand, nothingCommand)
+
+        assert(testCarGamePlayer.getPowerups().count(x => x == Config.LIZARD_POWERUP_ITEM) == 1);
+    }
+
+    test("Given players during race when player hits 2 lizard power ups then player gains 2 lizard powerups") {
```

Comment:
Is there a limit to how many lizard powerups a play can collect, without using any of them?

Reply:
I don't believe there are any limits to the amount of powerups players can pick up.

Reply:
Nah, these kinds of tests became the norm after the first bug we had when we launched. As for lizard there is no concern of "abuse" because you still interact with the world in the last block. 

That being said we may need to balance that generation chance if players can just collect lizards then jump their way to victory. But shouldn't be a problem given the other powerups.